docs/pages/reference/backends.mdx

```diff
@@ -13,7 +13,7 @@ read/write ratio, mutability, etc.).
 | Data type | Description | Supported storage backends |
 | - | - | - |
 | core cluster state | Cluster configuration (e.g. users, roles, auth connectors) and identity (e.g. certificate authorities, registered nodes, trusted clusters). | Local directory (SQLite), etcd, PostgreSQL, Amazon DynamoDB, GCP Firestore, CockroachDB |
-| audit events | JSON-encoded events from the audit log (e.g. user logins, RBAC changes) | Local directory, PostgreSQL, AWS DynamoDB, GCP Firestore |
+| audit events | JSON-encoded events from the audit log (e.g. user logins, RBAC changes) | Local directory, PostgreSQL/CockroachDB, AWS DynamoDB, GCP Firestore |
```

Comment:
For audit events, they are equal. For core backend they are not. There is no audit event cockroachdb backend, you need to use the postgres one.

Reply:
I'm not sure because in one case, the user needs enterprise + explicitly specify the cockroachDB backend. In the other, they configure a postgres audit backend that happens to be backed by crdb but it's transparent.